src/components/itemsCardsList.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ItemsCardsList from './itemsCardsList';
+import itemsData, { BASE_IMG_URL } from '../data/items';
+
+describe('ItemsCardsList', () => {
+  it('renders component', () => {
+    expect(itemsData.length).toBeGreaterThan(1);
```

Comment:
Testing your own constants is a bit unnecessary

Reply:
This is done to check that the test receives data for two or more cards.